objectstore.go

```diff
@@ -25,17 +25,23 @@ type ObjectStore struct {
 
 // CreateObjectStoreRequest holds the request to create a new object storage
 type CreateObjectStoreRequest struct {
-	Name       string `json:"name" validate:"required"`
-	MaxSizeGB  int    `json:"max_size_gb" validate:"required"`
-	MaxObjects int    `json:"max_objects"`
-	Region     string `json:"region"`
+	//Name            string `json:"name,omitempty" schema:"name"`
+	Name            string `json:"-"`
```

Comment:
I think it's fine if you enable it in civogo. We have it disabled on the API side. CLI flags can control when to send `Name`, so we can disable that flag